src/libraries/System.Security.Principal.Windows/ref/System.Security.Principal.Windows.cs

```diff
@@ -262,8 +262,10 @@ public partial class WindowsIdentity : System.Security.Claims.ClaimsIdentity, Sy
         public static System.Security.Principal.WindowsIdentity GetCurrent() { throw null; }
         public static System.Security.Principal.WindowsIdentity GetCurrent(bool ifImpersonating) { throw null; }
         public static System.Security.Principal.WindowsIdentity GetCurrent(System.Security.Principal.TokenAccessLevels desiredAccess) { throw null; }
-        public static void RunImpersonated(Microsoft.Win32.SafeHandles.SafeAccessTokenHandle safeAccessTokenHandle, System.Action action) { }
+        public static void RunImpersonated(Microsoft.Win32.SafeHandles.SafeAccessTokenHandle safeAccessTokenHandle, System.Action action) { throw null; }
```

Comment:
This doesn't need to throw as it returns void. It's harmless, but will be a diff when we regenerate at some point.

Reply:
ok understood